### PR TITLE
fix(autocomplete): :bug: fix autocomplete not showing the full page

### LINKED ISSUE #178: autocomplete cannot show the full choices page
### Describe the bug
When screen is shorter, autocomplete cannot show the full choices page.
Probably caused by the default page limit to be 10.
It could be better if calculate limit by the screen size.

screen record

### To Reproduce

1. The code
```js
const prompts = require('prompts');

(async function() {
  const { name } = await prompts({
    type: 'autocomplete',
    name: 'name',
    message: 'Select name',
    choices: Array.from({ length: 26 }, function(v, index) {
      return String.fromCharCode(index + 65);
    }),
    suggest(input, choices) {
      if (!input) {
        return choices;
      }
      return choices.filter(function(choice) {
        return choice.includes(input);
      });
    },
  });
  console.log('Selected name:', name);
})();
```

2. Run it.

### Expected behavior
See the full page by shorten the page size.

### System

- OS: MacOS 10.14.5
- Terminal: Terminal
- Node version: v11.11.0

### PR DESCRIPTION
**Pull request by vivaxy**
by accessing the window height, we set the default page size to height - 2 (1 for the question and
the other for the help)

closes #178

### CODE REVIEW

**Comment by terkelg** [comment]
Is there a way we can do this without introducing extra dependencies?

**Comment by vivaxy** [comment]
Removed extra dependency. @terkelg

File: lib/util/windowSize.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
+/**
+ * @since 2019-07-04 22:48:16
+ * @author vivaxy
+ */
+function get(stdout) {
+  const [width, height] = stdout.getWindowSize();
+  return { width, height };
+}
+
+function onChange(stdout, onChange) {
```
[the remark this answers is not in the document]
Comment (vivaxy) [reply lib/util/windowSize.js]: @DrunkenPoney Thanks for your review, looking forward to your pr being merged.

**Comment by vivaxy** [comment]
This PR is not a good approach for the issue. Closing.

**Pull request closed without merging**